.github/workflows/wheels.yml

```
@@ -24,15 +24,47 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  macos:
-    uses: ./.github/workflows/wheels-macos.yml
-    with:
-      artifacts-name: "dist"
-
-  linux:
-    uses: ./.github/workflows/wheels-linux.yml
-    with:
-      artifacts-name: "dist"
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: "macOS x86_64"
+            os: macos-latest
+            archs: x86_64
+          - name: "macOS arm64"
+            os: macos-latest
+            archs: arm64
+          - name: "manylinux2014 and musllinux x86_64"
+            os: ubuntu-latest
+            archs: x86_64
+          - name: "manylinux_2_28 x86_64"
+            os: ubuntu-latest
+            archs: x86_64
+            build: "*manylinux*"
+            manylinux: "manylinux_2_28"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+      - name: Build wheels
+        run: |
+          python3 -m pip install cibuildwheel
+          python3 -m cibuildwheel --output-dir wheelhouse
```

Perhaps we can use the action?
```suggestion
      - name: Build wheels
        uses: pypa/cibuildwheel@v2.16.2
        with:
          output-dir: wheelhouse
```

And https://cibuildwheel.readthedocs.io/en/stable/faq/#automatic-updates recommends pinning, whatever method is used:

> Selecting a moving target (like the latest release) is generally a bad idea in CI. If something breaks, you can't tell whether it was your code or an upstream update that caused the breakage, and in a worse-case scenario, it could occur during a release.

Renovate will keep it up-to-date.